purchase_blanket_order/views/purchase_blanket_order_views.xml

```diff
@@ -10,9 +10,11 @@
                 decoration-info="state in ('draft','to_approve')"
                 decoration-muted="state in ('expired')"
             >
+                <field name="partner_ref" optional="hide" />
                 <field name="name" />
                 <field name="user_id" />
                 <field name="partner_id" />
+                <field name="date_start" />
```

Comment:
Sorting now by this field should make that this one to be the first in the tree to have a logical distribution